libs/medic-users-meta/migrations/patch/202105171932.do.86.dropUsersMetaViews.sql

```diff
@@ -0,0 +1,7 @@
+-- Drop views with wrong indexes and definitions that were
+-- created at ../deprecated/202102191153.do.54-create-couchdb-users-meta-table.sql
+--
+-- The views are created again at ../202105171933.do.86.userMetaViews.sql
+
+DROP MATERIALIZED VIEW IF EXISTS useview_feedback;
```

Comment:
Can't we just run this patch for everyone and remove all the conditional code in the index file?
The `IF EXISTS` should cover situations where this didn't run right?

Reply:
Done.